context.go

```diff
@@ -204,6 +204,70 @@ func (module *moduleInfo) String() string {
 	return s
 }
 
+func (module *moduleInfo) GetBuildActions(c *Context) NinjaBuildActions {
```

Comment:
Will anything in Blueprint call this, or is it intended for the Soong unit tests?

Basically I wonder if we can refactor something Blueprint to use this to enforce that it remains up-to-date even if things change. If we have time, my inclination would be for ninja_writer.go#Build to take a NinjaBuildActions as an argument. It looks like ninja_defs.go might be related too.